pages/user-guides/[...chunks]/index.jsx

```diff
@@ -0,0 +1,320 @@
+import { useQuerySubscription } from 'utils/useQuerySubscription';
+import s from './style.module.css';
+
+import ChevronDown from 'public/icons/regular/chevron-down.svg';
+import ChevronLeft from 'public/icons/regular/chevron-left.svg';
+
+export const getStaticPaths = gqlStaticPaths(
+  `
+  {
+    chapters: allUserGuidesChapters {
+      slug
+      videos { slug }
+    }
+  }
+  `,
+  'chunks',
+  ({ chapters }) =>
+    chapters.flatMap((chapter) =>
+      chapter.videos.map((video) => [chapter.slug, video.slug]),
+    ),
+);
+
+export const getStaticProps = gqlStaticPropsWithSubscription(
+  /* GraphQL */ `
+    query itemQuery($chapterSlug: String!, $itemSlug: String!) {
+      item: userGuidesVideo(filter: { slug: { eq: $itemSlug } }) {
+        slug
+        seo: _seoMetaTags {
+          ...seoMetaTagsFields
+        }
+        title
+        video {
+          video {
+            playbackId: muxPlaybackId
+            duration
+          }
+          width
+          height
+        }
+        content {
+          value
+        }
+      }
+      currentChapter: allUserGuidesChapters(
+        filter: { slug: { eq: $chapterSlug } }
+      ) {
+        title
+        slug
+        videos {
+          title
+          slug
+          video {
+            video {
+              duration
+              thumbnailUrl
+              blurUpThumb
+              width
+              height
+            }
+          }
+          thumbTimeSeconds
+        }
+      }
+      otherChapters: allUserGuidesChapters(
+        filter: { slug: { neq: $chapterSlug } }
+      ) {
+        slug
+        title
+        videos {
+          slug
+        }
+      }
+      allChapters: allUserGuidesChapters {
+        slug
+        title
+        videos {
+          slug
+          video {
+            video {
+              duration
+              thumbnailUrl
+              blurUpThumb
+              width
+              height
+            }
+          }
+          thumbTimeSeconds
+        }
+      }
+    }
+
+    ${seoMetaTagsFields}
+  `,
+  {
+    requiredKeys: ['item', 'currentChapter[0]'],
+    paramsToVars: ({ chunks }) => ({
+      chapterSlug: chunks[0],
+      itemSlug: chunks[1],
+    }),
+  },
+);
+
+export default function Guide({ subscription, preview}) {
+  const {
+    data: { item, currentChapter, otherChapters, allChapters},
+  } = useQuerySubscription(subscription);
+
+  const currentChapterIndex = allChapters.findIndex(chapter => chapter.slug === currentChapter[0].slug);
+  const nextChapter = allChapters[currentChapterIndex + 1];
+  const getChapterIndexBySlug = (slug) => allChapters.findIndex(chapter => chapter.slug === slug) + 1;
+
+  const currentVideoIndex = currentChapter[0].videos.findIndex(video => video.slug === item.slug);
+  const nextVideo = currentChapter[0].videos[currentVideoIndex + 1];
+
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleAccordion = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
```

Comment:
sono confuso. qui sotto a volte controlli se item c'è (in vario modo, `item && ...`, oppure `item?.xxx`) altre volte invece ci accedi secco. questo mi fa pensare che item ci sia sempre, altrimenti si spaccherebbe nelle volte in cui ci accedi secco.

c'è da normalizzare la situazione, non solo su item, ma in generale, seguendo lo stesso ragionamento. se una cosa c'è sempre, rimuoviamo i conditionals che appesantiscono la lettura.